GsaGHTests/Helpers/GH/TextWrapperTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Reflection;
+
+using GsaGH.Helpers.GH;
+
+using Xunit;
+
+namespace GsaGHTests.Helpers {
+  [Collection("GrasshopperFixture collection")]
+  public class TextWrapperTests {
+    [Fact]
+    public void WrapTextShouldReturnEmptyWhenInputTextIsIsNullOrWhiteSpace() {
+      Assert.Empty(TextWrapper.WrapText(string.Empty, 1, 1));
+      Assert.Empty(TextWrapper.WrapText(null, 1, 1));
+      Assert.Empty(TextWrapper.WrapText("\n", 1, 1));
+      Assert.Empty(TextWrapper.WrapText(" ", 1, 1));
+    }
+
+    [Fact]
+    public void WrapText_ShouldWrapTextCorrectly() {
+      string inputText = "This is a test sentence that should be wrapped correctly.";
+      int maxWidth = 100;
+      int fontSize = 10;
+
+      string result = TextWrapper.WrapText(inputText, maxWidth, fontSize);
+
+      Assert.NotNull(result);
+      Assert.Contains("\n", result);
+      Assert.Equal("This is a test\nsentence that\nshould be\nwrapped\ncorrectly.", result);
+    }
```

Comment:
how did you figure what is the right result? I bet you had to run it once and whatever came out you use as "expected". 

I would have a test, where you put a really small `Width` let's say `1` and then put a text: `12` and then you expect to get `1\n2` which is understandable.

Then if you can make a more complex, where we can specify the length of the word and the length of the sentence by Mocking the algorithm then that is fine but it is secondary.